levels.c

```diff
@@ -0,0 +1,80 @@
+#include "levels.h"
+
+Level level;
+
+Brick create_brick(BrickType type, SpecificType capsule_reward) {
+    Brick brick;
+    brick.type = type;
+    brick.durability = 1;
+    brick.current_animation = 0;
+    brick.max_animation = 0;
+    brick.capsule_reward = capsule_reward;
+
+    if (type == METAL || type == GOLD) {
+        brick.durability = 8;
+        brick.max_animation = 6;
+    }
+
+    return brick;
+}
+
+void load_level(const char *filename) {
+    // Reset level
+    level.theme = THEME_1;
+    level.offset = 0;
+    level.height = 0;
+    for (int y = 0; y < LEVEL_HEIGHT; y++) {
+        for (int x = 0; x < LEVEL_WIDTH; x++) {
+            level.bricks[y][x] = create_brick(EMPTY, CAPSULE_EMPTY);
+        }
+    }
+
+    FILE *file = fopen(filename, "r");
+    if (file == NULL) {
+        fprintf(stderr, "Error: Could not open file %s\n", filename);
+        exit(1);
+    }
+
+    char line[8];
+
+    // Theme
+    fgets(line, sizeof(line), file);
+    level.theme = atoi(line);
+
+    // Offset
+    fgets(line, sizeof(line), file);
+    level.offset = atoi(line);
+
+    // Height
```

Comment:
I think you can remove those comments since we can read what the file line is about 🔥

Reply:
I'd rather not

Reply:
😒